src/cms/templates/pages/page_tree.html

```diff
@@ -117,6 +100,52 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
         </tbody>
     </table>
 </div>
+<div class="table-footer">
+    <div class="flex flex-wrap" style="padding-top: 0.6em;">
+        <div class="flex-wrap relative w-1/2">
+            <form class="table-search relative flex flex-wrap h-full">
+                <div class="pl-3" style="padding-top: 0.4em;">
+                    {% trans 'Bulk actions' %}
+                </div>
+                <div class="pl-3">
+                    <select class="h-full pl-4 pr-4 rounded shadow" style="padding-top: 5px;" name="bulk_action" id="bulk_action">
+                        <option>{% trans 'Please select' %}</option>
+                        <option value="archive_pages">{% trans 'Archive pages' %}
+                        {% for lang in languages %}
+                        {% if lang != region.default_language %}
+                        <option value="{{ lang.code }}">{% trans 'Export XLIFF for translation to' %} {{ lang.translated_name }}</option>
+                        {% endif %}
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="pl-3">
+                    <input type="button" onclick="bulk_action_execute();" class="inline-block cursor-pointer bg-blue-500 hover:bg-blue-600 text-white h-full font-bold py-2 px-4 rounded" value="{% trans 'Execute' %}" />
+                    <input type="hidden" id="bulk_action_url" value="{% url 'download_xliff' region_slug=region.slug language_code=language.code%}">
```

Comment:
I placed the URL in a hidden input to make it accessible to the compressed JS code. I think this is cleaner than having inline JS.